File: app/src/main/java/com/smith/ufc/comic/ComicRepository.java
package com.smith.ufc.comic;

import com.smith.ufc.data.base.Repository;
import com.smith.ufc.data.models.MarvelComic;
import com.smith.ufc.data.service.MarvelComicDataSource;

/**
 * Created by Charlton on 10/31/17.
 */

public interface ComicRepository extends Repository<MarvelComic, ComicSpecification>, MarvelComicDataSource {
}

File: app/src/main/java/com/smith/ufc/data/repositories/comic/ComicRepository.kt
package com.smith.ufc.data.repositories.comic

import com.smith.ufc.data.base.BaseRepository
import com.smith.ufc.data.models.MarvelComic
import com.smith.ufc.data.service.MarvelComicDataSource

/**
 * Created by Charlton on 10/31/17.
 */

interface ComicRepository : BaseRepository<MarvelComic, ComicSpecification>, MarvelComicDataSource
